.travis.yml

```diff
@@ -17,7 +17,7 @@ install:
 
 # command to run tests, e.g. python setup.py test
 script:
-    - python setup.py flake8
+    - flake8 OCA/openupgradelib
```

Comment:
it's weird, since the travis troubles this doesn't work any more: https://travis-ci.org/OCA/openupgradelib/jobs/147235136
I figure that's just fine too